src/QsFmt/Formatter/Rules.fs

```diff
@@ -101,3 +101,73 @@ let newLines =
             else
                 { block with CloseBrace = Terminal.mapPrefix ensureNewLine block.CloseBrace }
     }
+
+/// <summary>
+/// Gets the trivia from a terminal option.
+/// </summary>
+let getTrivia paren =
+    match paren with
+    | None -> []
+    | Some p -> p.Prefix
+
+let qubitBindingUpdate =
```

Comment:
There's a bug that causes a comment before the `)` in a using block to be duplicated:

```qsharp
namespace Foo {
    operation Foo() : Unit {
        using (
            q = Qubit() // comment
        ) {
        }
    }
}
```

turns into

```qsharp
namespace Foo {
    operation Foo() : Unit {
        use
            q = Qubit() // comment
         { // comment

        }
    }
}
```

I think it's probably related to using the context to store the prefix.

Reply:
I think this is fixed now with using the mapPrefix functions. I added this case as a unit test as well.